backend/msgpack/writer.go

```diff
@@ -34,6 +34,11 @@ import (
 	"github.com/uber-go/tally"
 )
 
+const (
+	buffersMetric = "buffers"
```

Comment:
nit: Given that these are only used in the `newWriterMetrics`, perhaps move these consts there?